entries/l/lu.ma.json

```
@@ -0,0 +1,12 @@
+{
+  "Luma": {
+    "domain": "lu.ma",
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://help.lu.ma/p/two-factor-authentication",
+    "categories": [
+      "tickets"
+    ]
```

```suggestion
    ],
    "regions": [
      "-cu",
      "-ir",
      "-kp",
      "-ru",
      "-sy"
    ]
```

See the Export Controls and Designated Persons section of the [Terms of Use](https://lu.ma/terms).